### ISSUE TITLE
mistaken files in the published 0.10.0?

### ISSUE THREAD

**Issue by cuviper** [bytecodealliance/target-lexicon#41]
I noticed some files in the published `target-lexicon-0.10.0.crate` that might not be intended:

```text
$ tar tvf target-lexicon-0.10.0.crate
-rw-rw-r-- 1001/1001        70 2018-04-27 14:33 target-lexicon-0.10.0/.gitignore
-rw-rw-r-- 1001/1001         0 2018-04-27 14:30 target-lexicon-0.10.0/.rustfmt.toml
-rw-rw-r-- 1001/1001      1261 2019-09-05 05:42 target-lexicon-0.10.0/.travis.yml
-rw-rw-r-- 1001/1001       597 2019-11-18 21:09 target-lexicon-0.10.0/Cargo.toml.orig
-rw-r--r-- 0/0            1119 1969-12-31 16:00 target-lexicon-0.10.0/Cargo.toml
-rw-rw-r-- 1001/1001     12243 2018-10-30 12:41 target-lexicon-0.10.0/LICENSE
-rw-rw-r-- 1001/1001       957 2019-08-27 16:15 target-lexicon-0.10.0/README.md
-rw-rw-r-- 1001/1001      5593 2019-11-18 21:09 target-lexicon-0.10.0/build.rs
-rw-rw-r-- 1001/1001       208 2019-10-04 17:05 target-lexicon-0.10.0/examples/host.rs
-rw-rw-r-- 1001/1001       405 2019-11-05 10:16 target-lexicon-0.10.0/examples/misc.rs
-rw-rw-r-- 1001/1001      1399 2019-03-06 14:11 target-lexicon-0.10.0/src/host.rs
-rw-rw-r-- 1001/1001       877 2019-11-18 21:09 target-lexicon-0.10.0/src/lib.rs
-rw-rw-r-- 1001/1001      1208 2019-11-04 07:24 target-lexicon-0.10.0/src/parse_error.rs
-rw-rw-r-- 1001/1001     44839 2019-11-18 21:09 target-lexicon-0.10.0/src/targets.rs
-rw-rw-r-- 1001/1001     12722 2019-11-18 21:09 target-lexicon-0.10.0/src/triple.rs
-rw-rw-r-- 1001/1001      1878 2019-11-05 08:38 target-lexicon-0.10.0/a
-rw-rw-r-- 1001/1001      1532 2019-11-18 21:12 target-lexicon-0.10.0/host.rs
-rwxrwxr-x 1001/1001       212 2019-06-13 01:47 target-lexicon-0.10.0/newlist
-rw-rw-r-- 1001/1001      5592 2019-11-18 21:12 target-lexicon-0.10.0/sorted.txt
-rwxrwxr-x 1001/1001       367 2019-11-18 21:12 target-lexicon-0.10.0/test.sh
-rw-r--r-- 0/0             147 1969-12-31 16:00 target-lexicon-0.10.0/Cargo.lock
```

In particular, the last 5 -- `a`, `host.rs`, `newlist`, `sorted.txt`, and `test.sh` -- are not in the repo.

**Comment by sunfishcode** [comment]
Oops, thanks for spotting this! I'll fix this in the next release.

**Comment by sunfishcode** [comment]
This is fixed in 0.11.0.
